File: FlightControl/Models/Models/Flight.cs
using System.ComponentModel.DataAnnotations;

namespace Models.Models
{
    public class Flight : BaseEntity
    {

        [Range(2, 30)]
        public string NameOfСhiefPilot { get; set; }

        [Required]
        [Range(2,30)]
        public string From { get; set; }

        [Required]
        public int Speed { get; set; }

        [Required]
        public int NumberOfPass { get; set; }

        public int Fuel { get; set; }

        public string Image { get; set; }

        public int DistanceToTerminal { get; set; }

        public PlaneTerminalState PlaneTerminalState { get; set; }

    }
    public enum PlaneTerminalState
    {
        In_Air_1=1,
        In_Air_2,
        In_Air_3,
        Landing_4,
        Arrivals_Stock_5,
        Вisembarking_Boarding_Passengers_6,
        Вisembarking_Boarding_Passengers_7,
        Departurs_Stock_8,
        Takeoff_9,
        InAirTakeoff_10,
        Non
    }
}

File: angular-client/src/app/models/flight.ts
export class Flight{

    id:any;    
    nameOfСhiefPilot:string;
    from :string
    speed:number
    numberOfPass:number;
    fuel :number;
    image :string;
    planeTerminalState: PlaneTerminalState
    distanceToTerminal:number;
}
export enum PlaneTerminalState
{
    In_Air_1=1,
    In_Air_2,
    In_Air_3,
    Landing_4,
    Arrivals_Stock_5,
    Вisembarking_Boarding_Passengers_6,
    Вisembarking_Boarding_Passengers_7,
    Departurs_Stock_8,
    Takeoff_9,
    Non
}